scripts/setup-gcloud-cluster.sh

```diff
@@ -200,45 +167,43 @@ for instance_idx in "${!instance_names_arr[@]}" ; do
   wait_until_node_is_ready "${node_ip}"
 
   if [[ "${reset_metadata}" = "true" ]]; then
-    gcloud_remove_instance_metadata "${instance_name}" "HOPRD_PEER_ID,HOPRD_WALLET_ADDR,HOPRD_STAKING_ADDR"
+    gcloud_remove_instance_metadata "${instance_name}" "HOPRD_PEER_ID,HOPRD_WALLET_ADDR,HOPRD_STAKING_STATUS"
   fi
 
   # All VM instances in the deployed cluster will get a special metadata entries
   # which contain all information about the HOPR instance running in the VM.
   # These currently include:
   # - node wallet address
   # - node peer ID
-  # - associated staking account
+  # - staking status
   # This information is constant during the lifetime of the VM and
   # does not change during re-deployment once set.
   declare instance_metadata
   instance_metadata="$(gcloud_get_node_info_metadata "${instance_name}")"
 
   # known metadata keys
-  declare wallet_addr peer_id staking_addr
+  declare wallet_addr peer_id staking_status
   wallet_addr="$(echo "${instance_metadata}" | jq -r '."HOPRD_WALLET_ADDR" // empty')"
   peer_id="$(echo "${instance_metadata}" | jq -r '."HOPRD_PEER_ID" // empty')"
-  staking_addr="$(echo "${instance_metadata}" | jq -r '."HOPRD_STAKING_ADDR" // empty')"
+  staking_status="$(echo "${instance_metadata}" | jq -r '."HOPRD_STAKING_STATUS" // empty')"
```

Comment:
The changes in these metadata attributes have an impact on the GCP Packer image, as those metadata are being read by Fluentbit to publish attributes to Graylog. I will create a separate PR in hopr-devops where it removes the HOPRD_STAKING_STATUS